<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadHelper</class>
 <widget class="QWidget" name="DownloadHelper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DownloadHelper</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Target Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditDir"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item>
      <widget class="QToolButton" name="tBtnChooseDir">
       <property name="text">
        <string>choose dir</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditDir</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>tBtnChooseDir</tabstop>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
